scheduler/test/cook/test/kubernetes/compute_cluster.clj

```diff
@@ -66,7 +67,7 @@
     (let [pod-name->pod {"podA" (pod-helper "podA" "hostA" {:cpus 1.0 :mem 100.0})}]
       (is (= {"hostA" {:cpus 1.0
                        :mem 100.0}}
-             (kcc/get-consumption pod-name->pod)))))
+             (api/get-consumption pod-name->pod)))))
```

Comment:
These tests should be in a different namespace, since they're just testing code in `api`

Reply:
Moved.